shell/browser/api/electron_api_web_contents.h

```diff
@@ -488,6 +488,7 @@ class WebContents final : public ExclusiveAccessContext,
 
   // content::WebContentsDelegate:
   bool IsWebContentsCreationOverridden(
+      content::RenderFrameHost* opener,
```

Comment:
Looks like this was needed because of https://chromium-review.googlesource.com/c/chromium/src/+/6513812.